### PR TITLE
docs(roadmap): fix headings, update str0m, webtransport, autonatv2 & address pipeline

### PR DESCRIPTION
**Pull request by mxinden**
## Notes & open questions

Here are the updates suggested in the last open maintainers call. Any objections?

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -8,31 +8,33 @@ This is a living document. Input is always welcome e.g. via GitHub issues or pul
 This is the roadmap of the Rust implementation of libp2p. See also the [general libp2p project
 roadmap](https://github.com/libp2p/specs/blob/master/ROADMAP.md).
 
-## Attempt to switch from webrtc-rs to str0m
+## TODO and in-progress
```
Comment (thomaseizinger) [inline ROADMAP.md]:
```suggestion
## In the works
```

Sounds a bit nicer maybe :)

File: ROADMAP.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ roadmap](https://github.com/libp2p/specs/blob/master/ROADMAP.md).
...
-| Connectivity | todo   |                   | https://github.com/libp2p/rust-libp2p/issues/3659 |              |            |
+| Category                    | Status | Target Completion | Tracking                                          | Dependencies                       | Dependents |
+|-----------------------------|--------|-------------------|---------------------------------------------------|------------------------------------|------------|
+| Connectivity / optimization | todo   |                   | https://github.com/libp2p/rust-libp2p/issues/2993 | [QUIC](#experimental-quic-support) |            |
 
-Reduce maintenance burden and reduce dependency footprint.
+A WebTransport implementation in rust-libp2p will enable browsers to connect to rust-libp2p nodes
+where the latter only have a self-signed TLS certificate. Compared to WebRTC, this would likely be
+more performant. It is dependent on QUIC support in rust-libp2p. Given that we will support WebRTC
+(browser-to-server) this is not a high priority.
```
Comment (thomaseizinger) [inline ROADMAP.md]: Mind switching to one sentence per line?

Plus, I given that we have QUIC now, I think we line about the QUIC dependency can be removed.

Comment (thomaseizinger) [reply ROADMAP.md]: Also, the line about the priority seems to be outdated now :)

**Review by thomaseizinger — APPROVED** [review]
No objections but some suggestions!

File: ROADMAP.md (hunk 1 above)
Comment (mxinden) [reply ROADMAP.md]: Much better. Thank you @thomaseizinger.

**Comment by mxinden** [comment]
Thanks @thomaseizinger.

**Comment by thomaseizinger** [comment]
@mxinden Your PR title exceeds the maximum allowed length.

**Comment by mxinden** [comment]
> @mxinden Your PR title exceeds the maximum allowed length.

Shall I mark it as _required_ in https://github.com/libp2p/github-mgmt @thomaseizinger?

**Pull request merged**